### PR TITLE
Modal slider

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Modal And Slider
## Added :
- Modal component
- Slider component

### CODE REVIEW

File: Components/Modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+              style="
+                background-color: #fafafa;
```
Comment (teja538) [inline Components/Modal/modal.html:133]: small suggestion : you can use separate class for iframe styling and can reuse it.

**Review by teja538 — COMMENTED** [review]
code looks good and clean
looks good to merge

File: Components/Slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.slider-container {
+  gap: 10px;
+  align-items: center;
+}
+.slider-range {
+  appearance: none;
+  background-color: var(--silder-color);
```
Comment (devdeadviz) [inline Components/Slider/slider.css:7]: There is a typo in your variable naming. I think it should be `slider-color`

**Review by devdeadviz — APPROVED** [review]
The code is clean, and you have followed all the best practices. Good Work!

File: Components/Modal/modal.html (hunk 1 above)
Comment (dusariudaykumar) [reply Components/Modal/modal.html:133]: Yeah, I tried but it's breaking the entire frame

File: Components/Slider/slider.css (hunk 2 above)
Comment (dusariudaykumar) [reply Components/Slider/slider.css:7]: Sure i will fix it

**Pull request merged**